parachain/runtime/interlay/src/lib.rs

```diff
@@ -1073,6 +1081,7 @@ construct_runtime! {
         Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 6,
         Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 7,
         Identity: pallet_identity::{Pallet, Call, Storage, Event<T>} = 8,
+        Sudo: pallet_sudo::{Pallet, Call, Storage, Config<T>, Event<T>} = 9,
```

Comment:
Actually, I believe this can be index 2 as that is where is was before when we previously had it enabled on mainnet before launch.

Reply:
Slightly annoying we either way will have different indexes between interlay and kintsugi

Reply:
I will make the change in https://github.com/interlay/interbtc/pull/693